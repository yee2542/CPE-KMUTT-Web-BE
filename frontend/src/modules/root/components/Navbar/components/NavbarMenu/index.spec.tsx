import * as React from 'react';

import { shallow } from 'enzyme';

import 'jest-styled-components';

import * as NavbarMenuType from '.';

describe('Navbar Tests', () => {
  const { default: NavbarMenu } = require('.') as typeof NavbarMenuType;
  it('Navbar menu should be defined', () => {
    const wrapper = shallow(<NavbarMenu />);
    expect(wrapper).toMatchSnapshot();
  });
});
