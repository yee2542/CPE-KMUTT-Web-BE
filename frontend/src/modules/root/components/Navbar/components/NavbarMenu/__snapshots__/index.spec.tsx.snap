// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Tests Navbar menu should be defined 1`] = `
<styled__MenuWrap>
  <Container>
    <styled__Menu>
      <styled__Group
        key="1"
      >
        <styled__Image
          img="https://cpe.kmutt.ac.th/uploadFile/facebook-share-cover.jpg"
        >
          <styled__Mask />
        </styled__Image>
        <styled__Item>
          ปริญญาตรี
        </styled__Item>
        <styled__ItemLink
          href="/#"
          key="1"
        >
          หลักสูตรปกติ
        </styled__ItemLink>
        <styled__ItemLink
          href="/#"
          key="2"
        >
          หลักสูตรนานาชาติ
        </styled__ItemLink>
        <styled__ItemLink
          href="/#"
          key="3"
        >
          หลักสูตรวิทยาศาสตร์สุขภาพ
        </styled__ItemLink>
        <styled__ItemLink
          href="/#"
          key="4"
        >
          หลักสูตร Residental College
        </styled__ItemLink>
      </styled__Group>
      <styled__Group
        key="2"
      >
        <styled__Image
          img="https://cpe.kmutt.ac.th/uploadFile/facebook-share-cover.jpg"
        >
          <styled__Mask />
        </styled__Image>
        <styled__Item>
          บัณฑิตศึกษา
        </styled__Item>
        <styled__ItemLink
          href="/#"
          key="1"
        >
          หลักสูตรมหาบัณฑิต
        </styled__ItemLink>
        <styled__ItemLink
          href="/#"
          key="2"
        >
          หลักสูตรดุษฎีบัณฑิต
        </styled__ItemLink>
      </styled__Group>
      <styled__Group
        key="3"
      >
        <styled__Image
          img="https://cpe.kmutt.ac.th/uploadFile/facebook-share-cover.jpg"
        >
          <styled__Mask />
        </styled__Image>
        <styled__Item>
          ทุนการศึกษา
        </styled__Item>
        <styled__ItemLink
          href="/#"
          key="1"
        >
          ทุนภาควิชา
        </styled__ItemLink>
        <styled__ItemLink
          href="/#"
          key="2"
        >
          ทุนเพชรพระจอมเกล้า
        </styled__ItemLink>
        <styled__ItemLink
          href="/#"
          key="3"
        >
          ทุน คปก.
        </styled__ItemLink>
        <styled__ItemLink
          href="/#"
          key="4"
        >
          ทุนต่างประเทศ
        </styled__ItemLink>
      </styled__Group>
      <styled__Group
        key="4"
      >
        <styled__Image
          img="https://cpe.kmutt.ac.th/uploadFile/facebook-share-cover.jpg"
        >
          <styled__Mask />
        </styled__Image>
        <styled__Item>
          การรับเข้าศึกษา
        </styled__Item>
        <styled__ItemLink
          href="/#"
          key="1"
        >
          รับตรง (Active Recruitment)
        </styled__ItemLink>
        <styled__ItemLink
          href="/#"
          key="2"
        >
          รับส่วนกลาง
        </styled__ItemLink>
        <styled__ItemLink
          href="/#"
          key="3"
        >
          โครงการพิเศษ
        </styled__ItemLink>
      </styled__Group>
      <styled__Group
        key="5"
      >
        <styled__Image
          img="https://cpe.kmutt.ac.th/uploadFile/facebook-share-cover.jpg"
        >
          <styled__Mask />
        </styled__Image>
        <styled__Item>
          เกี่ยวกับเรา
        </styled__Item>
        <styled__ItemLink
          href="/#"
          key="1"
        >
          ภาควิชา
        </styled__ItemLink>
        <styled__ItemLink
          href="/#"
          key="2"
        >
          บุคลากร
        </styled__ItemLink>
        <styled__ItemLink
          href="/#"
          key="3"
        >
          นักศึกษา
        </styled__ItemLink>
        <styled__ItemLink
          href="/#"
          key="4"
        >
          ความเป็นนานาชาติ
        </styled__ItemLink>
      </styled__Group>
    </styled__Menu>
    <styled__SubMenu>
      <styled__SubMenuItem
        key="1"
      >
        สำหรับนักศึกษาปัจจุบัน
      </styled__SubMenuItem>
      <styled__SubMenuItem
        key="2"
      >
        สำหรับบุคลากร
      </styled__SubMenuItem>
      <styled__SubMenuItem
        key="3"
      >
        ดาวน์โหลด
      </styled__SubMenuItem>
      <styled__SubMenuItem
        key="4"
      >
        ติดต่อเรา
      </styled__SubMenuItem>
    </styled__SubMenu>
  </Container>
</styled__MenuWrap>
`;
